### DISCUSSION TITLE
Exclude micronaut-grpc-bom from gradle dependencies

### DISCUSSION THREAD

**Discussion by eratolekov** [micronaut-projects/micronaut-core#11111] [Q&A]
Hi there,

Is it possible to exclude `micronaut-grpc-bom` from gradle dependencies?
Micronaut 4.5

I have collision with grpc because
```
\--- io.micronaut.platform:micronaut-platform:4.5.1
     +--- io.micronaut.grpc:micronaut-grpc-bom:4.5.0
     |    +--- io.grpc:grpc-bom:1.64.0
```

**Comment by eratolekov** [comment]
Discussion 11111 😁

**Reply by graemerocher** [reply]
I believe you can disable the BOM in the `micronaut` block and apply it manually excluding gRPC
